src/backend/expungeservice/endpoints/users.py

```diff
@@ -110,6 +110,62 @@ def post(self):
         return jsonify(response_data), 201
 
 
+    @user_auth_required
+    def put(self, user_id):
+        """
+        Update the user entry with new values for one or more of the user data fields:
+        email, name, group_name, password, or admin
+        """
+        user_db_data = user.read(g.database, user_id)
+        if not user_db_data:
+            error(404, "User id not recognized")
+
+        if not g.logged_in_user_is_admin and g.logged_in_user_id != user_id:
+            error(403, "Logged in user not admin and doesn't match requested user id.")
+
+
+        data = request.get_json()
+
+        if data is None:
+            error(400, "No json data in request body")
+
+        if not any([key in data.keys() for key in ["email", "name", "group_name",
+                                 "password", "admin"]]):
+
+            error(400, "Json data must define one or more of: \
+email, name, group_name, password, admin")
```

Comment:
I don't see a reason to check for this. 1. if there is no data being updated it wont change anything anyway. 2. we can ensure that there is always data by passing the user data to the edit form on the frontend. That way they can see what is currently in the user profile, and make changes as necessary. Of course there is no harm in checking for this, however I don't think it needs to be here.